src/components/SideRow.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./SideRow.css";
+import {NavLink} from "react-router-dom";
+let activeStyle = {
+  textDecoration: "none",
+  color: "white",
+  backgroundColor: "var(--sidebar-hover-color) ",
+};
+export function SideRow({title, Icon, to}) {
+  return (
+    <NavLink
+      end
+      className={`mg-left-2 btn-secondary-md flex row align-items-center siderow`}
+      to={`${to}`}
```

Comment:
If you're not forming a URL then just use the variable directly. Assuming that the passed value of `to` is always a `String`

```suggestion
to={to}
```